src/routes/__tests__/ArticleChoiceExerciseScreen.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import { RouteProp } from '@react-navigation/native'
```

Comment:
Should be in `routes/choice-exercises/__tests__` folder.

Also why have you only added a Test for ArticelChoiceExerciseScreen, but not for WordChoiceExerciseScreen? Would be nice if you can add also a test for this.

Reply:
sorry i forgot the uni test when i created pr so i would have to revert this pr to draft, since it wasn't finished so